Integrations/WindowsDefenderAdvancedThreatProtection/WindowsDefenderAdvancedThreatProtection.py

```diff
@@ -801,26 +799,34 @@ def get_alert_related_user(alert_id):
 
 def fetch_incidents():
     last_run = demisto.getLastRun()
+    last_fetch_str = last_run.get('last_alert_fetched_time')
+    previous_ids = last_run.get('last_ids', [])
 
-    if last_run and last_run['last_alert_fetched_time']:
-        last_alert_fetched_time = datetime.strptime(last_run['last_alert_fetched_time'], '%Y-%m-%dT%H:%M:%S.%f')
-    else:
-        last_alert_fetched_time = datetime.now() - timedelta(days=300)
+    if not last_fetch_str:
+        last_fetch_str, _ = parse_date_range(FETCH_TIME, date_format='%Y-%m-%dT%H:%M:%S.%fZ')
 
-    previous_ids = last_run.get('last_ids', [])
-    latest_creation_time = last_alert_fetched_time
+    date_filter = f'alertCreationTime gt {last_fetch_str}'
+    severity_filter = f'severity in ({FETCH_SEVERITY})'
+    status_filter = f'status in ({FETCH_STATUS})'
+
+    raw_filter = f'{date_filter} and {severity_filter} and {status_filter}'
+
+    params = {
+        '$top': MAX_FETCH_RESULTS,
+        '$skip': '0',
+        '$filter': raw_filter
+    }
+
+    alerts = list_alerts(params)['value']
 
-    alerts = list_alerts()['value']
     incidents = []
     last_ids = []
 
+    last_fetch = datetime.strptime(last_fetch_str, '%Y-%m-%dT%H:%M:%S.%fZ')
+    latest_creation_time = last_fetch
     for alert in alerts:
-        # Removing 'Z' from timestamp and converting to datetime
-        alert_creation_time = datetime.strptime(alert['alertCreationTime'][:-2], '%Y-%m-%dT%H:%M:%S.%f')
-        alert_status = alert['status']
-        alert_severity = alert['severity']
-        if alert_creation_time >= last_alert_fetched_time and alert_status in FETCH_STATUS and \
-                alert_severity in FETCH_SEVERITY and alert['id'] not in previous_ids:
+        alert_creation_time = datetime.strptime(alert['alertCreationTime'], '%Y-%m-%dT%H:%M:%S.%fZ')
+        if alert_creation_time > last_fetch and alert['id'] not in previous_ids:
```

Comment:
i think that you've deleted the status and severity consideration for the fetch mech